api/router.py

```diff
@@ -0,0 +1,32 @@
+import shortuuid
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm import Session
+
+import models
+from database import get_db
+
+router = APIRouter(
+    prefix="/api",
+    tags=["Api"]
+)
+
+
+@router.post("/create-short-link")
+def create_short_link(link: str, db: Session = Depends(get_db)) -> str:
```

Comment:
you can add validation to the link using pydantics's type: HttpUrl

Reply:
It's also better to pass the link through the body, it's a more canonical look for the post request